src/screens/QrScanner.js

```diff
@@ -207,3 +212,45 @@ const styles = StyleSheet.create({
     paddingBottom: 20,
   }
 });
+
+function rawDataToU8A(rawData) {
+  if (!rawData) {
+    return null;
+  }
+
+  // Strip SQRC filler bytes
+  while (rawData.substr(-4) === 'ec11') {
+    rawData = rawData.substr(0, rawData.length - 4);
```

Comment:
sometimes the padding ends with `ec11ec`, so this will not always work. e.g. `4370000010000530100f4cd755672a8f9542ca9da4fbf2182e79135d94304002e6a09ffc96fef6e6c4c5448495320495320535041525441210ec11ec11ec11ec` for payload `THIS IS SPARTA!`